src/templates/home-page.js

```diff
@@ -84,7 +84,8 @@ export const HomePageTemplate = ({
           <h1 className="section-title">Latest News</h1>
           <div className="row">
             { newsItems && newsItems.map(({ node }) => (
-              <div className="latest-news-item col-sm-12 col-md-4" key={node.id}>
+              <div className="latest-news-item col-sm-12 col-lg-4" key={node.id}>
+                <div className="date">{node.frontmatter.date}</div>
```

Comment:
Worth noting that you could use a `time` element here, though I don't think it's a big issue.